docs/make.bat

```diff
@@ -28,6 +29,9 @@ if errorlevel 9009 (
 %SPHINXBUILD% -M %1 %SOURCEDIR% %BUILDDIR% %SPHINXOPTS% %O%
 goto end
 
+:livehtml
+sphinx-autobuild %SOURCEDIR% %BUILDDIR% %SPHINXOPTS% %O%
+
```

Comment:
This needs a `goto end`, otherwise it'll continue into the `:help` section which I don't think is intentional.

Reply:
Thanks, missed that one :+1: